memtagabielf64/README.md

```diff
@@ -17,19 +17,18 @@ is licensed under CC-BY-SA-4.0 along with an additional patent
 license.  The language in the additional patent license is largely
 identical to that in Apache-2.0 (specifically, Section 3 of Apache-2.0
 as reflected at https://www.apache.org/licenses/LICENSE-2.0) with two
-exceptions.
-
-First, several changes were made related to the defined terms so as to
-reflect the fact that such defined terms need to align with the
-terminology in CC-BY-SA-4.0 rather than Apache-2.0 (e.g., changing
-“Work” to “Licensed Material”).
-
-Second, the defensive termination clause was changed such that the
-scope of defensive termination applies to “any licenses granted to
-You” (rather than “any patent licenses granted to You”).  This change
-is intended to help maintain a healthy ecosystem by providing
-additional protection to the community against patent litigation
-claims.
+exceptions:
+
+- First, several changes were made related to the defined terms so as to reflect
+the fact that such defined terms need to align with the terminology in
+CC-BY-SA-4.0 rather than Apache-2.0 (e.g., changing “Work” to “Licensed
+Material”).
+
+- Second, the defensive termination clause was changed such that the scope of
+defensive termination applies to “any licenses granted to You” (rather than “any
+patent licenses granted to You”).  This change is intended to help maintain a
+healthy ecosystem by providing additional protection to the community against
+patent litigation claims.
```

Comment:
These lines need indenting to be inline with "S" in "Second".

Reply:
see first comment